docs/standard/library-guidelines/breaking-changes.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Breaking Changes
+description: Best practice recommendations for navigating breaking changes when creating .NET libraries.
+author: jamesnk
+ms.author: jamesnk
+ms.date: 09/20/2018
+---
+# Breaking Changes
+
+It's important for an open-source project to find a balance between stability for existing users and innovation for the future. As developers, we lend towards refactoring and rethinking code until it's perfect, but breaking your existing users has a negative impact, especially for low-level libraries.
+
+## Project types and breaking changes
+
+How a project is used by the .NET community changes the effect of breaking changes on end-user developers.
+
+* **Low and middle-level libraries** like a serializer, HTML parser, DB ORM, or web framework are the most effected by breaking changes.
```

Comment:
Because you mention serializers here, you may want to define a data format breaking change. It may not apply because the library could manage that, but it feels like an omission.

Reply:
I don't follow. I think the example paragraph immediately below this is long enough without an additional example.

Reply:
Sorry, you define "source breaking", behavior breaking", and "binary breaking" changes further down in the document. A serialize library could also introduce a data format change, where persistent data from a previous version can't be read using a newer version. It may be a distraction, in which case let's leave it out, but I wanted to make sure you thought about it.

Reply:
I added changing data output as an example of behavior breaking change lower in the doc.

There are so many way to introduce behavior breaking changes that it isn't feasible to go into detail.